dll/shellext/netshell/shfldr_netconnect.cpp

```diff
@@ -641,6 +641,14 @@ HRESULT WINAPI CNetConUiObject::InvokeCommand(LPCMINVOKECOMMANDINFO lpcmi)
     {
         CmdId = IDS_NET_PROPERTIES;
     }
+    else if ((SIZE_T)lpcmi->lpVerb == FCIDM_SHVIEW_RENAME) // DefView accelerator
```

Comment:
MAKEINTRESOURCE(...lpVerb) or ULONG_PTR cast, instead of the size_t cast (which doesn't cast a size), + same comment for below.

Reply:
MAKEINTRESOURCE casts to string, not what I want.


> instead of the size_t cast (which doesn't cast a size)

malloc takes a size_t so clearly it is a size. But ok, a pointer type is slightly more correct.